Examples/Supervised/Ising/ed.py

```diff
@@ -33,9 +33,8 @@ def load_ed_data(L):
     ttargets = []
 
     tsamples = []
-    hind = nk.hilbert.HilbertIndex(hi)
-    for i in range(hind.n_states):
-        tsamples.append(hind.number_to_state(i).tolist())
+    for i in range(hi.index.n_states):
```

Comment:
This is nice! but what would be **really cool** is an iterator

```python
for state in hi.states():
   tsample.append(state)
```

Reply:
Agreed. I can give it a try.